.bazelrc

```diff
@@ -6,4 +6,8 @@ build:windows --compiler="clang-cl" --cxxopt="/std:c++17"
 
 build:macos --cxxopt=-std=c++17
 
-build:linux --cxxopt=-std=c++17
\ No newline at end of file
+build:linux --config=clang_config --cxxopt=-std=c++17
+
+build:clang_config --crosstool_top=//toolchain:clang_suite
+build:clang_config --cpu=k8
```

Comment:
If I'm reading [this](https://docs.bazel.build/versions/main/cc-toolchain-config-reference.html) correctly we shouldn't need to specify the CPU because Bazel will pick the automatically select the cpu based on where we're currently running. It says "If no flags are specified, Bazel inspects the host system and selects a --cpu value based on its findings." So I agree we need `k8` down in the `cc_toolchain_suite` but doesn't seem like we need to specify it here and we might want to specify `piii` down in `cc_toolchain_suite` too (or leave a comment that since it hasn't been tested we didn't add it but may add it in the future.

Also, for posterity, [here's](https://cs.opensource.google/bazel/bazel/+/1b73bc37e184e71651eb631223dcce321ba16211:src/main/java/com/google/devtools/build/lib/analysis/config/AutoCpuConverter.java) the code that detects and selects the cpu.